tensorflow/compiler/mlir/hlo/lib/Dialect/gml_st/IR/gml_st_ops.cc

```diff
@@ -1823,14 +1824,15 @@ void SetYieldOp::build(
     builder.setInsertionPointToStart(&bodyBlock);
     (*builderFnIt)(builder, result.location, bodyBlock.getArgument(0),
                    bodyBlock.getArgument(1));
-    std::next(builderFnIt);
+    (void)std::next(builderFnIt);
```

Comment:
I'm confused, isn't this hiding a real bug here? (@pifon2a )

Reply:
See: https://godbolt.org/z/Kx3aezonj

Reply:
@joker-eph Yes, it is a real bug. It was not visible because we have never built a terminator with multiple accumulators programmatically. Thank you! I sent the fix.

Reply:
@ashay can you remove all these (void) for now?

Reply:
Sounds good, removed.